dotcom-rendering/src/web/components/Avatar.tsx

```diff
@@ -1,39 +1,107 @@
 import { css } from '@emotion/react';
+import { Fragment } from 'react';
 import type { DCRContainerPalette } from '../../types/front';
 import type { Palette } from '../../types/palette';
 import { decidePalette } from '../lib/decidePalette';
+import { generateSources } from './Picture';
 
-const contributorImage = css`
+const picture = css`
 	border-radius: 100%;
+	height: 100%;
+	width: 100%;
+	overflow: hidden;
+`;
+
+const img = css`
 	object-fit: cover;
 	height: 100%;
 	width: 100%;
 `;
 
+/**
+ * Used on `picture` and `img` to prevent having a line-height,
+ * as these elements are which are `inline` by default.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#styling_with_css
+ */
+const block = css`
+	display: block;
+`;
+
 const backgroundStyles = (palette: Palette) =>
 	css`
 		background-color: ${palette.background.avatar};
 	`;
 
 type Props = {
-	imageSrc: string;
-	imageAlt: string;
+	src: string;
+	alt: string;
 	format: ArticleFormat;
 	containerPalette?: DCRContainerPalette;
 };
 
-export const Avatar = ({
-	imageSrc,
-	imageAlt,
-	format,
-	containerPalette,
-}: Props) => {
+/**
+ * # TEMPORARY MEASURE
+ *
+ * We can handle generating the image URL ourselves,
+ * so we’re turning signed “i.guim.co.uk” URLs into
+ * their source ones
+ *
+ * @deprecated we should update frontend to send sources directly
+ */
+const getSourceImage = (src: string) => {
+	const url = new URL(src);
+	if (url.hostname !== 'i.guim.co.uk') return src;
+
+	const [, , domain] = url.pathname.split('/');
+	if (!['uploads', 'media'].includes(domain)) return src;
+
+	return new URL(
+		url.pathname.replace(`/img/${domain}/`, ''),
+		`https://${domain}.guim.co.uk`,
+	).toString();
+};
```

Comment:
This is a bit unfortunate, but we had to have a similar approach when we first introduced `CardPicture`, as the image paths coming from `frontend` are already signed.